### PR TITLE
#37 add integration mails

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 1]
```diff
@@ -1,53 +1,46 @@
...
-      "Iz-maWBTKYHMeQmxB",
-    )
-      .then((response) => {
-        console.log("SUCCESS!", response.status, response.text);
-      })
-      .catch((err) => {
-        console.log("FAILED...", err);
-      });
-    e.target.reset();
+    setBookingFormData(defaultBookingFormData);
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx:33]: what happened here? why do we need to set default booking form data?

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 2]
```diff
@@ -71,8 +64,8 @@ export const Form = (props: RoomProps) => {
             className={styles.input}
             id="email"
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx:65]: why do we need id here?

File: gf-webapp/src/logic/contactsPage/ContactsPage.tsx [hunk 3]
```diff
@@ -1,45 +1,29 @@
 import React, {useRef, useState} from "react";
 import {useDictionary} from "../DictionaryContext/useDictionary";
-import {send} from "emailjs-com";
 import {PageBorder} from "../../component/pageBorder/PageBorder";
+import {MailApiService} from "src/service/MailApi/MailApi";
+import {ApiAskAdminPostRequest} from "src/autogenerated/gf-server-api/models/ApiAskAdminPostRequest";
 import styles from "./ContactsPage.module.scss";
 import {Button} from "gf-ui-lib/components/Button/Button";
 
+const defaultAskAdminFormData: ApiAskAdminPostRequest = {name: "", email: "", phone: "", message: ""};
```
Comment (tritonJS826) [inline gf-webapp/src/logic/contactsPage/ContactsPage.tsx:9]: bad format - set linter to put props on new lines

File: gf-webapp/src/logic/contactsPage/ContactsPage.tsx [hunk 4]
```diff
@@ -1,45 +1,29 @@
...
-      "Iz-maWBTKYHMeQmxB",
-    )
-      .then((response) => {
-        console.log("SUCCESS!", response.status, response.text);
-      })
-      .catch((err) => {
-        console.log("FAILED...", err);
-      });
-    e.target.reset();
+    setAskAdminFormData(defaultAskAdminFormData);
```
Comment (tritonJS826) [inline gf-webapp/src/logic/contactsPage/ContactsPage.tsx:21]: why do we need set default data on submit?

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

**Review by scorpigg — APPROVED** [review]

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 1 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx:33]: Deleted set default values in inputs

File: gf-webapp/src/logic/contactsPage/ContactsPage.tsx (hunk 4 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/contactsPage/ContactsPage.tsx:21]: Deleted set default values in inputs

File: gf-webapp/src/logic/contactsPage/ContactsPage.tsx (hunk 3 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/contactsPage/ContactsPage.tsx:9]: done

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx:65]: Replaced input inside label and deleted id

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**